components/proxy/qualitytime.conf.template

```diff
@@ -0,0 +1,35 @@
+server {
+    listen       ${NGINX_LISTEN};
+    server_name  localhost;
```

Comment:
This file seems to be a copy of `default.conf.template` (with these lines added) but the `access_log      off;` directive has been dropped. It would be preferable to template all functionalities within a single file